@app/db/scripts/dump-db.js

```diff
@@ -0,0 +1,31 @@
+const { spawn } = require("child_process");
+
+const inTests = process.env.IN_TESTS;
+if (!inTests) {
+  const connectionString = process.env.GM_DBURL;
+  if (!connectionString) {
+    throw new Error(
+      "This script should only be ran from inside graphile-migrate"
+    );
```

Comment:
No need for a stack trace here.

```suggestion
    console.error(
      "This script should only be called from a graphile-migrate action."
    );
    process.exit(1);
```